### ISSUE TITLE
[Accessibility][CommandBar]: Disabled menuitem must have aria-disabled true

### ISSUE THREAD

**Issue by shyama1208** [OfficeDev/office-ui-fabric-react#1114]
Hi,
We are using the CommandBar to show buttons such as "Save", "Queue", etc. The buttons support enable/disable states.
We found that the disabled button/menuitem does not have aria-disabled property set to true.
This is required as per ARIA design pattern for menuitem.
 https://www.w3.org/TR/wai-aria-practices-1.1/#button

![image](https://cloud.githubusercontent.com/assets/26054709/23351290/3f04c936-fce6-11e6-83ab-5461add49a5b.png)

Requesting you to fix this issue as appropriate.

Thanks,
Shyama Shukla.

**Comment by mdahamiwal** [comment]
This change is being done in #942.

**Comment by gitjain** [comment]
Addressed in #942 will be reopened if necessary
